src/core/hle/kernel/thread.cpp

```diff
@@ -409,16 +415,43 @@ void Thread::BoostPriority(u32 priority) {
 
 std::shared_ptr<Thread> SetupMainThread(KernelSystem& kernel, u32 entry_point, u32 priority,
                                         std::shared_ptr<Process> owner_process) {
+
+    constexpr const s64 sleep_random_deviation_ns = 250'000'000LL;
+    constexpr const s64 sleep_normal_thread_base_ns = 250'000'000LL;
+    constexpr const s64 sleep_app_thread_base_ns = 2'500'000'000LL;
+    constexpr const u32 system_module_tid_high = 0x00040130;
```

Comment:
```suggestion
    constexpr s64 sleep_random_deviation_ns = 250'000'000LL;
    constexpr s64 sleep_normal_thread_base_ns = 250'000'000LL;
    constexpr s64 sleep_app_thread_base_ns = 2'500'000'000LL;
    constexpr u32 system_module_tid_high = 0x00040130;
```
const is not necessary with constexpr